specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/Product.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "billingAccountId": "1234",
+    "invoiceSectionId": "42425",
+    "productName": "productSummaryId1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": {
+          "id": "/providers/Microsoft.Billing/BillingAccounts/1234/products/productSummaryId1",
+          "name": "productSummaryId1",
+          "type": "Microsoft.Consumption/products",
+          "properties": {
+            "productTypeId": "A12345",
```

Comment:
Having a property "productTypeId" and "productType" implies the latter is an actual (expanded) resource. I believe it's simply a localized name, in which case I'd argue productType should be called e.g. productTypeName.